### PR TITLE
updated xcode version for Travis

### PR DESCRIPTION
**Pull request by JohnCremona**
Let's see if this works for Travis...

### CODE REVIEW

**Comment by JohnCremona** [comment]
This failed: the osx configure failed to find ntl.  Help would be appreciated.

**Comment by JohnCremona** [comment]
@isuruf do you know what is wrong here?  The configure script cannot find ntl when Travis runs.

**Comment by isuruf** [comment]
Looks like OSX works, but linux fails. Can you print `config.log`?

**Comment by JohnCremona** [comment]
@isuruf you are right I misread the output notification.  I don't know how to recover the file config.log from travis.

**Comment by JohnCremona** [comment]
I don't understand this. I had assumed that it was failing because of the publicised end of support for xcode 6.4 but as you saw it is the Linux build which fails.  Is it possible that the latest version of miniconda puts NTL somewhere different?  The error from running configure is that it cannot find NTL.  But nothing to do with NTL has changed.

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ os:
   - osx
   - linux
 
-osx_image: xcode6.4
+osx_image: xcode9.4
 dist: trusty
```
Comment (isuruf) [inline .travis.yml]: Can you change this to xenial? Linux error is because the NTL package was built with a newer compiler than what it is in trusty.

**Comment by JohnCremona** [comment]
Changing from trusty to xenial did the trick.  Thanks for suggesting that.

**Pull request merged**